templates/environment/cf.yml

```diff
@@ -437,6 +470,106 @@ Resources:
             ]
             Resource: "*"
 
+  # DNS Delegation Resources
+  CertificateValidationFunction:
```

Comment:
Can you check from the logs how much memory the fn is using now? To make sure we're not close to 128